shopware/meteor
# fix toast positioning

## What?

The toast notifications were cut off on smaller viewports.

## Why?

This hindered users from interacting with the toast component on small viewports.

## How?

I adjusted the flexbox positioning.

## Testing?

* Visit the branch preview below
* Make the viewport smaller
* Fire a new toast and check if it's fully visible
